pkgs/development/libraries/science/math/osqp/default.nix

```diff
@@ -17,6 +17,12 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
+  patches = [
+    # Fix header path including prefix twice.
+    # https://github.com/osqp/osqp/pull/481
+    ./fix-cmake-header-path.diff
```

Comment:
We should fetchpatch https://github.com/osqp/osqp/commit/e9dd8ed1b2943733010307cf3671ddde0e9a5ba6.patch